docs/docs/30-how-to-guides/14-working-with-stages.md

```diff
@@ -0,0 +1,548 @@
+        - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+          id: 961cfaedbc53aacdb65110028839a2c1c281290d
+          repoURL: https://github.com/example/kargo-demo.git
+        images:
+        - digest: sha256:b2487a28589657b318e0d63110056e11564e73b9fd3ec4c4afba5542f9d07d46
+          repoURL: public.ecr.aws/nginx/nginx
+          tag: 1.27.0
+        name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+        origin:
+          kind: Warehouse
+          name: my-warehouse
+    verificationHistory:
+    - analysisRun:
+        name: test.01j2w7aknhf3j7jteyqs72hnbg.101bca5
+        namespace: kargo-demo-09
+        phase: Successful
+      finishTime: "2024-07-15T22:13:57Z"
+      id: 5535a484-bbd0-4f12-8cf4-be2c8e0041c9
+      phase: Successful
+      startTime: "2024-07-15T22:13:34Z"
+  health:
+    argoCDApps:
+    - healthStatus:
+        status: Healthy
+      name: kargo-demo-09-test
+      namespace: argocd
+      syncStatus:
+        revision: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+        status: Synced
+    status: Healthy
+  lastPromotion:
+    finishedAt: "2024-07-15T22:13:25Z"
+    freight:
+      commits:
+      - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+        id: 961cfaedbc53aacdb65110028839a2c1c281290d
+        repoURL: https://github.com/example/kargo-demo.git
+      name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+      origin:
+        kind: Warehouse
+        name: kargo-demo
+    name: test.01j2w7a15cxjjgejresfyw6ysp.666209f
+    status:
+      finishedAt: "2024-07-15T22:13:25Z"
+      freight:
+        commits:
+        - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+          id: 961cfaedbc53aacdb65110028839a2c1c281290d
+          repoURL: https://github.com/example/kargo-demo.git
+        name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+        origin:
+          kind: Warehouse
+          name: kargo-demo
+      freightCollection:
+        id: 101bca5b0e18ca7913978a1da956308d2544f741
+        items:
+          Warehouse/kargo-demo:
+            commits:
+            - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+              id: 961cfaedbc53aacdb65110028839a2c1c281290d
+              repoURL: https://github.com/example/kargo-demo.git
+            name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+            origin:
+              kind: Warehouse
+              name: kargo-demo
+        verificationHistory:
+        - analysisRun:
+            name: test.01j2w7aknhf3j7jteyqs72hnbg.101bca5
+            namespace: kargo-demo-09
+            phase: ""
+          id: 5535a484-bbd0-4f12-8cf4-be2c8e0041c9
+          phase: Pending
+          startTime: "2024-07-15T22:13:34Z"
+      phase: Succeeded
+  observedGeneration: 1
+  phase: Steady
+```
+
+## Interacting with Stages
+
+Kargo provides tools to manage Stages using either the Kargo Dashboard or the
+Kargo CLI. This section explains how to handle Stages effectively through both interfaces.
```

Comment:
There are a lot of places throughout the docs where we say UI or dashboard. I think we should standardize on one term. We'll have to clean it up in other places as well, but let's get started here.

```suggestion
Kargo provides tools to manage Stages using either its UI or
CLI. This section explains how to handle Stages effectively through both interfaces.

:::info
Users with credentials for and sufficient permissions within the Kargo control plane's Kubernetes cluster can also manage `Stage` resources using `kubectl`.
:::
```